frontend/src/app/commonComponents/Header.jsx

```diff
@@ -30,12 +83,13 @@ const Header = ({ organizationId }) => {
             "is-visible": menuVisible,
           })}
         >
-          <button
-            className="usa-nav__close prime-nav-close-button"
+          <span
```

Comment:
This should stay a button for a11y/508 reasons. You can style it to look however you need though.

Reply:
Agreed. Fixed in both places.